components/layout.js

```diff
@@ -12,23 +12,45 @@ import QuickLook from './quick-look'
 
 const prefix = 'https://images.carbonplan.org'
 
-const Tool = ({
-  title,
-  description,
-  meta,
-  contentWidth = [6, 10],
-  descriptionWidth = [6, 6, 6, 6],
-  quickLookStart = 8,
-  children,
-}) => {
+const meta = {
+  id: 'compliance-users',
+  title: 'Compliance users',
+  color: 'pink',
```

Comment:
style nit: this is the color used for facility -- do we need a fourth color?

Reply:
trying orange! looks nice color pairing wise, only concern is we've used it for soils...